doc/examples/plot_adapt_rgb.py

```diff
@@ -48,7 +48,9 @@ def sobel_hsv(image):
 
 fig, (ax_each, ax_hsv) = plt.subplots(ncols=2)
 ax_each.imshow(sobel_each(image))
+ax_each.set_title('Each channel separately')
 ax_hsv.imshow(sobel_hsv(image))
+ax_hsv.set_title('Value filtered channel')
```

Comment:
How about "Filtered on HSV value channel"? I think "value" is clearer if you mention HSV as well.